sdcm/sla/sla_tests.py

```diff
@@ -147,11 +154,12 @@ class SlaTests(Steps):
     def unique_subsrtr_for_name():
         return str(uuid.uuid1()).split("-", maxsplit=1)[0]
 
-    @staticmethod
-    def _create_sla_auth(session, shares: int, index: str, superuser: bool = True) -> Role:
+    # pylint: disable=too-many-arguments
+    def _create_sla_auth(self, session, tester, shares: int, index: str, superuser: bool = True) -> Role:
```

Comment:
better provide straight `db_cluster` instead of `tester` - as we don't need `tester`

Reply:
changed